pkg/cmd/roachtest/main.go

```diff
@@ -284,6 +287,8 @@ runner itself.
 			&count, "count", 1, "the number of times to run each test")
 		cmd.Flags().BoolVarP(
 			&debugEnabled, "debug", "d", debugEnabled, "don't wipe and destroy cluster if test fails")
+		cmd.Flags().BoolVar(
+			&skipInit, "skip-init", false, "skip initialization step (imports, table creation, etc.) for tests, useful when re-using clusters with --wipe=false")
```

Comment:
Done. I kept the `skip-init` name since the binary name is different (`roachtest`) and hopefully a clear enough demarcation (+ wanted to avoid the stuttering in `roachtest --skip-test-init`).